zerver/webhooks/trello/view/card_actions.py

```diff
@@ -48,7 +56,9 @@
     REMOVE_MEMBER: u'removed {member_name} from {card_url_template}',
     ADD_ATTACHMENT: u'added [{attachment_name}]({attachment_url}) to {card_url_template}',
     ADD_CHECKLIST: u'added the {checklist_name} checklist to {card_url_template}',
-    COMMENT: u'commented on {card_url_template}'
+    MOVE_UP: u'moved {card_url_template} up in the list {list}',
+    MOVE_DOWN: u'moved {card_url_template} down in the list {list}',
+    COMMENT: u'commented on {card_url_template}\n>{text}\n'
 }
```

Comment:
The newline `\n` at the end of:
```
COMMENT: u'commented on {card_url_template}\n>{text}\n'
```
leads to notifications such as the one below, with a stray period after every quoted comment:

![stray_period_trello](https://user-images.githubusercontent.com/7251823/30517288-c05b4e28-9b34-11e7-8601-631156e70706.png)

Removing that newline at the end would fix that!

Also, I think it would be clearer if you used `~~~ quote <text> ~~~` in the `COMMENT` template instead of `>{text}`. Its a lot more explicit and is the syntax that a lot of our webhook tests use. See [here for an example](https://github.com/zulip/zulip/blob/master/zerver/lib/webhooks/git.py#L39). :)